src/cmd/validate/validate.go

```diff
@@ -0,0 +1,74 @@
+package validate
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/defenseunicorns/go-oscal/src/pkg/validation"
+	"github.com/spf13/cobra"
+)
+
+type flags struct {
+	InputFile string // -f --input-file
+	LogFile   string // -l --logger-file
+
+}
+
+var opts = &flags{}
+
+var ValidateCmd = &cobra.Command{
+	Use:   "validate",
+	Short: "validate an oscal document",
+	Long:  "Validate an OSCAL document against the OSCAL schema version specified in the document.",
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Do you have any opinions on the use of `Run` vs `RunE` in cobra? Potentially useful for returning errors. Generate already uses `RunE` but that doesn't mean it's fully utilizing what is exposed. 

https://gopheradvent.com/calendar/2022/taming-cobras-making-most-of-cobra-clis/#:~:text=RunE%20is%20identical%20to%20Run,code%20inside%20the%20main%20function.

Reply:
Yeah I think it may be more useful in a pipeline etc... If we use RunE and return the errors. I don't think setting it without that in mind will change much. I can switch it here or create a story for command tests that should expose the usefulness of ```RunE```, or both.

Reply:
Definitely - I'd love to see errors being exposed up to the command in such a way that we can do greater handling in the future. IE a nicety of RunE is that it returns an error which automatically invoked the command help message. That native functionality can go a long way for UX. 

Let's write a follow-on issue for that modification so as to not block this unnecessarily.

Reply:
https://github.com/defenseunicorns/go-oscal/issues/55